src/Raven.Server/Documents/Indexes/Persistence/Lucene/Documents/AnonymousLuceneDocumentConverter.cs

```diff
@@ -92,7 +93,7 @@ protected override int GetFields<T>(T instance, LazyStringValue key, LazyStringV
                     throw new InvalidOperationException($"Field '{property.Key}' is not defined. Available fields: {string.Join(", ", _fields.Keys)}.", e);
                 }
 
-                var numberOfCreatedFields = GetRegularFields(instance, field, value, indexContext, out var shouldSkip);
+                var numberOfCreatedFields = GetRegularFields(instance, field, value, indexContext, out var shouldSkip, sourceDocument);
```

Comment:
Can we put the new parameter before `out` param? It's easier to notice the our parameters when they are put as the last ones

Reply:
done